### DISCUSSION TITLE
Welcome to sci-cpy Discussions!

### DISCUSSION THREAD

**Discussion by Dutta-SD** [Dutta-SD/sci-cpy#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ZenithFlux** [comment]
Hey Sandip, sorry to bother you again but this pull request was not registered in hacktober website. Just asking, if you can accept the same pull request if i send it again so that hacktober can record it?

**Reply by Dutta-SD** [reply]
Hi
So as far as I know, Hacktoberfest takes a bit of time to record a pull request once it is merged. So let's wait for some time. 

If still it does not happen, I will be happy to once again accept your pull request. 

Just make a new PR with some minor change. I will merge it in the repo. :smile:

**Reply by ZenithFlux** [reply]
Actually, i have confirmed from some of my friends that hacktober website was down for sometime at the time you accepted my PR. Maybe that's why it didn't record the merge.

I will wait till tomorrow morning and if website still doesn't show it, I will create a PR.

Thank you for your support sandip!![Screenshot_20211001-180927_Chrome.jpg](https://user-images.githubusercontent.com/90706514/135620787-dafb1545-f6ff-436f-97c1-8c13549e4531.jpg)

**Reply by Dutta-SD** [reply]
I merged your PR now. Do let me know if everything was alright! Happy Hacktoberfest!

**Reply by ZenithFlux** [reply]
Yeah it worked this time.

This was a really unique program to make.
I had a great time working on this issue.

Thanks and happy hacktoberfest!

![Screenshot_20211002-095924_Chrome.jpg](https://user-images.githubusercontent.com/90706514/135703602-3099b198-723e-4aea-ab40-87b22a114bfc.jpg)
